packages/Babylonian-UI.package/BPBrowser.class/instance/initializeSidePaneFrom..st

```diff
@@ -0,0 +1,4 @@
+initialize-release
+initializeSidePaneFrom: aBrowser
+
+	sidePane := aBrowser submorphNamed: self sidePaneNameSymbol.
```

Comment:
is the variable necessary if you can access this morph with the `submorphNamed:` method?